src/test/java/ladder/domain/player/NameTest.java

```diff
@@ -0,0 +1,27 @@
+package ladder.domain.player;
+
+import ladder.exception.TooLongNameException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class NameTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"aaaaaa" , "bbbbbbbbbbbbbbbbb", "ccccccccccccccccccccccc"})
+    @DisplayName("이름이 5글자가 넘으면 예외를 던진다.")
+    public void TooLongNameExceptionTest(String name) {
+        Assertions.assertThrows(TooLongNameException.class , () ->
+                new Name(name)
+        );
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"kim:kim", "hong:hong", "pobi:pobi"}, delimiter = ':')
+    public void NameTest(String name, String result)
+    {
+        org.assertj.core.api.Assertions.assertThat(new Name(name).toString()).isEqualTo(result);
+    }
```

Comment:
```suggestion
  @ParameterizedTest
  @ValueSource(strings = {"kim", "hong", "pobi"})
  public void NameTest(String text) {
    Name name = new Name(text);
    Assertions.assertEquals(name.toString(), text);
  }
```
이렇게 작성해보면 어떨까요?

한 라인에 모든 내용을 넣기보다는 줄 바꿈을 이용해서 가독성을 높였으면 좋겠어요.
그리고 테스트 코드는 [Given - When - Then](https://brunch.co.kr/@springboot/292) 구조로 작성해보시는 연습을 해보는 것도 좋을 것 같네요. 😄

Reply:
다시보니 해당 코드는 가독성이 낮긴했었네요 :)